site/training/training.qmd

```diff
@@ -40,39 +41,39 @@ listing:
     - path: https://jupyterhub.validmind.ai/hub/user-redirect/lab/tree/quickstart_customer_churn_full_suite.ipynb
       title: "Quickstart for model documentation"
       subtitle: "Open notebook in JupyterHub {{< fa chevron-right >}}"
-      description: "Gets you started with the basic process of documenting models with ValidMind, from the {{< var vm.developer >}} to the {{< var vm.platform >}}."
+      description: "Gets you started with the basic process of documenting models with {{< var vm.product >}}, from the {{< var vm.developer >}} to the {{< var vm.platform >}}."
       categories: ["[Demo] Customer Churn Model"]
       reading-time: "10"
-      author: "ValidMind"
+      author: "{{< var vm.product >}}"
     - path: https://jupyterhub.validmind.ai/hub/user-redirect/lab/tree/tutorials/intro_for_model_developers.ipynb
-      title: "ValidMind introduction for model developers"
+      title: "{{< var vm.product >}} introduction for model developers"
       subtitle: "Open notebook in JupyterHub {{< fa chevron-right >}}"
       description: "Learn how the end-to-end documentation process works based on common scenarios you encounter in model development settings."
       categories: ["[Demo] Customer Churn Model"]
       reading-time: "27"
-      author: "ValidMind"
+      author: "{{< var vm.product >}}"
     - path: https://jupyterhub.validmind.ai/hub/user-redirect/lab/tree/code_samples/credit_risk/application_scorecard_demo.ipynb
       title: "Document an application scorecard model"
       subtitle: "Open notebook in JupyterHub {{< fa chevron-right >}}"
       description: "Guides you through building and documenting an application scorecard model using the Lending Club sample dataset from Kaggle."
       categories: ["[Demo] Credit Risk Model"]
       reading-time: "16"
-      author: "ValidMind"
+      author: "{{< var vm.product >}}"
     - path: https://jupyterhub.validmind.ai/hub/user-redirect/lab/tree/code_samples/nlp_and_llm/foundation_models_integration_demo.ipynb
       title: "Prompt validation for large language models (LLMs)"
       subtitle: "Open notebook in JupyterHub {{< fa chevron-right >}}"
       description: "Run and document prompt validation tests for a large language model (LLM) specialized in sentiment analysis for financial news."
       categories: ["[Demo] Foundation Model - Text Sentiment Analysis"]
       reading-time: "8"
-      author: "ValidMind"
+      author: "{{< var vm.product >}}"
 ---
 
 ::: {.column-screen}
 
 ::: {.training-hero}
 
 :::: {.f1 .w-60}
-**ValidMind Academy**
+**{{< var vm.product >}} Academy**
```

Comment:
Same here, I'd omit the variable or use a new variable for the training program name.